### ISSUE TITLE
[V1.2-> Scenarios - Expense Report.json]: 'Export As PDF' link is not accessible with Voiceover using Swipe gestures in forward and backward both navigation.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#6983]
https://microsoft.visualstudio.com/OS/_workitems/edit/35693107
